### PR TITLE
[gen] Add tests for CodeTerm

### PR DESCRIPTION
**Pull request by c-harding**

### CODE REVIEW

**Review by jaylees14 — APPROVED** [review]
Looks good to moi

File: src/main/scala/temple/generate/utils/CodeTerm.scala [hunk 1]
```diff
@@ -106,9 +106,12 @@ object CodeTerm {
     /** Add a string before the opening symbol, without any spacing */
     def prefix(prefix: String): CodeWrap = new CodeWrap(prefix + start, end)
 
-    /** Wrap a (comma-separated list of) terms in parentheses */
+    /** Wrap a (space-separated list of) terms in parentheses */
     def apply(string: CodeTerm*): String = mkCode(start, string, end)
 
+    /** Wrap a (comma-separated list of) terms in parentheses */
+    def list(string: CodeTerm*): String = mkCode(start, mkCode.list(string), end)
```
Comment (willjvsmith) [inline src/main/scala/temple/generate/utils/CodeTerm.scala:113]: boo now I got to make changes again

**Review by willjvsmith — APPROVED** [review]

File: src/main/scala/temple/generate/utils/CodeTerm.scala (hunk 1 above)
Comment (willjvsmith) [reply src/main/scala/temple/generate/utils/CodeTerm.scala:113]: Should also probs mention this stuff in PR desc, since it's not mentioned in title

Comment (c-harding) [reply src/main/scala/temple/generate/utils/CodeTerm.scala:113]: Sorry, noticed it was missing when I saw the comment on apply was wrong

**Pull request merged**